active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,234 @@
+# Number box
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Number box is a numerics only text control with support for validation, increment stepping, and computing inline calculations.
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox) [TODO - generated with MSDN documentation on publication.]
+
+## Is this the right control? 
+
+Use a **NumberBox** to capture and display mathematical input. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across parenthetical order with standard operator precedence.
```

Comment:
Have we validated that these operators will be sufficient? What about exponents? Percentage? I think we should support anything that the Cortana numberbox calculator can support.

Reply:
As a graphics application example...
If you had a value of **256 px**, how would you enter the operator to find the 25% value of that original value?

### 256 25%

Or do we mentally figure out that 25% is dividing the value by 4?

### 256 / 4

Reply:
Good question. I don't have a strong opinion on percentage specifically; my meta-feedback is that NumberBox will have a list of supported operators, and I just wanted to ensure that we were being thoughtful about which operators we support. (The percent sign can also be modulo, so that adds a potential level of complexity to this specific operator.)

Reply:
Initiated required operators conversation up-level on the feature proposal. Closing here to maintain a single source thread. Thanks @stmoy and @mdtauk for starting this!